src/main/java/org/bukkit/craftbukkit/v1_20_R3/entity/CraftEntityTypes.java

```diff
@@ -380,6 +261,9 @@ Level minecraftWorld() {
         // None spawn able
         register(new EntityTypeData<>(EntityType.FISHING_HOOK, FishHook.class, CraftFishHook::new, null)); // Cannot spawn a fish hook
         register(new EntityTypeData<>(EntityType.PLAYER, Player.class, CraftPlayer::new, null)); // Cannot spawn a player
+        //Ketting: Todo: Proper EntityType?
```

Comment:
What do I put here?

Reply:
No clue, i did not place that there

Reply:
Needs some intervention from ForgeInject probably. idk